docs/developer-guide/mapstore-migration-guide.md

```diff
@@ -107,6 +107,45 @@ Following actions need to be applied to make a switch:
 - Remove "BurgerMenu" entry from "desktop" section.
 
 #### Updating contexts to use Sidebar Menu
+Contents of your `pluginsConfig.json` need to be reviewed to allow usage of new "SidebarMenu" in contexts.
+- Find "BurgerMenu" plugin confuguration in `pluginsConfig.json` and remove `"hidden": true` line from it:
+```json
+    {
+  "name": "BurgerMenu",
+  "glyph": "menu-hamburger",
+  "title": "plugins.BurgerMenu.title",
+  "description": "plugins.BurgerMenu.description",
+  "dependencies": [
+    "OmniBar"
+  ]
+}
+```
+- Add "SidebarMenu" entry to the "plugins" array:
+```json
+{
+    "plugins": [
+        ...
+        {
+            "name": "SidebarMenu",
+            "hidden": true
+        }
+        ...
+    ]
+}
+```
+- Go through all plugins definitions and replace "BurgerMenu" dependency with "SidebarMenu", e.g.:
+```json
+    {
+      "name": "MapExport",
+      "glyph": "download",
+      "title": "plugins.MapExport.title",
+      "description": "plugins.MapExport.description",
+      "dependencies": [
+        "SidebarMenu"
+      ]
+    }
+```
```

Comment:
There are some error in rendring of documentation
(see here:  https://mapstore--8085.org.readthedocs.build/en/8085/developer-guide/mapstore-migration-guide/#replacing-burgermenu-with-sidebarmenu  )

Probably there is no space before the list element ( ` - Find Burger menu ....`)

````suggestion

Contents of your `pluginsConfig.json` need to be reviewed to allow usage of new "SidebarMenu" in contexts.

- Find "BurgerMenu" plugin confuguration in `pluginsConfig.json` and remove `"hidden": true` line from it:

```json
    {
  "name": "BurgerMenu",
  "glyph": "menu-hamburger",
  "title": "plugins.BurgerMenu.title",
  "description": "plugins.BurgerMenu.description",
  "dependencies": [
    "OmniBar"
  ]
}
```

- Add `"SidebarMenu"` entry to the "plugins" array:

```json
{
    "plugins": [
        ...
        {
            "name": "SidebarMenu",
            "hidden": true
        }
        ...
    ]
}
```

- Go through all plugins definitions and replace `"BurgerMenu"` dependency with `"SidebarMenu"`, e.g.:

```json
    {
      "name": "MapExport",
      "glyph": "download",
      "title": "plugins.MapExport.title",
      "description": "plugins.MapExport.description",
      "dependencies": [
        "SidebarMenu"
      ]
    }
```

- Also the `StreetView` plugin needs to depend from `SidebarMenu`. 

```json
{
      "name": "StreetView",
      "glyph": "road",
      "title": "plugins.StreetView.title",
      "description": "plugins.StreetView.description",
      "dependencies": ["Toolbar"]
      "dependencies": [
        "SidebarMenu"
      ]
}
```

````

Reply:
Thanks! I've applied your changes with a fix to line 208 above